SampleOpenGL.py

```diff
@@ -0,0 +1,368 @@
+#
+# Copyright 2022 NVIDIA Corporation
```

Comment:
Copyright shall belong to the author ;)
This sample wasn't developed by NVIDIA.

Reply:
Ah!

Reply:
@Yves33 and @sandhawalia 

Please add your copyright(s) here.

Reply:
@Yves33 and @sandhawalia 

Please add your copyright(s) here.